pkg/server/router/schema.go

```diff
@@ -34,8 +34,12 @@ type CreateSchemaRequest struct {
 	Author string               `json:"author" validate:"required"`
 	Name   string               `json:"name" validate:"required"`
 	Schema schemalib.JSONSchema `json:"schema" validate:"required"`
+
 	// Sign represents whether the schema should be signed by the author. Default is false.
+	// If sign is true, the schema will be signed by the author's private key with the specified KID
 	Sign bool `json:"sign"`
+	// AuthorKID represents the KID of the author's private key to sign the schema. Required if sign is true.
+	AuthorKID string `json:"authorKid"`
```

Comment:
why did this not work like the other APIs where its the DID and we would pull the key out of keystore to sign?

Reply:
Since a DID can have multiple keys associated with it, we won't have a way of knowing which key to use without a KID specified.